slice/IceGrid/Admin.ice

```diff
@@ -753,9 +753,8 @@ module IceGrid
     /// @see Registry
     interface AdminSession extends Glacier2::Session
     {
-        /// Keep the session alive. Clients should call this operation regularly to prevent the server from reaping the
-        /// session.
-        /// @see Registry#getSessionTimeout
+        /// Keep the session alive. This operation is provided for backwards compatibility. As of Ice 3.8, there is no
```

Comment:
Keeping this operation and making it no-op is nicer for backwards compatibility.

Reply:
We should deprecate the operation.

Reply:
I tried and it turns out the deprecated metadata is broken: #2085.